blob/fileblob/fileblob.go

```diff
@@ -693,6 +693,7 @@ func (h *URLSignerHMAC) URLFromKey(ctx context.Context, key string, opts *driver
 	q.Set("obj", key)
 	q.Set("expiry", strconv.FormatInt(time.Now().Add(opts.Expiry).Unix(), 10))
 	q.Set("method", opts.Method)
+	q.Set("contentType", opts.ContentType)
```

Comment:
The fileblob tests fail with this change, I think you need to verify the content type in the handler?